telegram/ext/filters.py

```diff
@@ -2545,3 +2548,74 @@ def filter(self, message: Message) -> bool:
 
 VOICE = _Voice("filters.VOICE")
 """Messages that contain :attr:`telegram.Message.voice`."""
+
+
+class Mention(MessageFilter):
+    """Mention messages. If list of username or chat_id or `User` objects passed, it filters
+    messages to only allow those whose contains one of username  or chat_id or `User`.
+    Note:
+        Usernames only allowed with @ prefix. Example: "@username"
+        Chat id only allowed with positive numbers. Example: 123456
+    Examples:
+        A simple use case for passing username is to allow only messages that were sent with
+        mention username:
+        MessageHandler(filters.Mention("@username"), callback_method)
+        MessageHandler(filters.Mention(["@username",`123456`,`user`), callback_method)
```

Comment:
```suggestion
        MessageHandler(filters.Mention(["@username",  123456, user), callback_method)
```

Let's pass chat_ids as integers and not as strings.